src/videojs.ads.js

```diff
@@ -1,3 +1,13 @@
+(function(factory){
+    "use strict";
+    if (typeof define === 'function' && define.amd) {
+        define(['videojs'], function(vjs){factory(window, document, vjs);});
```

Comment:
Oops. Yeah, it makes sense as a dependency. I'm inclined to go with a named module so we don't run into the concatenation problem in downstream dependencies again.